webrtc-c/canary/src/Cloudwatch.h

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+namespace Canary {
+
+class Synchronization {
+  public:
+    std::atomic<bool> pending;
+    std::recursive_mutex mutex;
+    std::condition_variable_any await;
+};
+
+class CloudwatchLogs {
+  public:
+    CloudwatchLogs(Canary::PConfig, ClientConfiguration*);
+    STATUS init();
+    VOID deinit();
+    VOID push(string log);
+    VOID flush(BOOL sync = FALSE);
+
+  private:
+    PConfig pConfig;
+    CloudWatchLogsClient client;
+    Synchronization sync;
+    Aws::Vector<InputLogEvent> logs;
+    Aws::String token;
+};
+
+class CloudwatchMonitoring {
```

Comment:
Perhaps split this into it's own header like the source?

Reply:
Yep, makes sense